hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -527,7 +528,7 @@ private OzoneManager(OzoneConfiguration conf, StartupOption startupOption)
           omStorage.getOmId(), omNodeDetails.getRpcAddress(),
           omNodeDetails.getRatisPort());
     }
-
+    this.threadPrefix = omNodeDetails.threadNamePrefix();
```

Comment:
`private final String threadPrefix` and `private String threadNamePrefix` are still duplicated.

Reply:
Sorry about that! I have made the changes.
Thanks!